ReservacionVuelos/src/components/TextTerms.js

```diff
@@ -0,0 +1,16 @@
+import {StyleSheet, Text} from 'react-native';
+import React from 'react';
+import signUpStyles from '../utils/styles/signUpStyles';
+
+const TextTerms = () => {
+  return (
+    <Text>
+      I agree to the
+      <Text style={signUpStyles.textTerms}>Terms and Privacy Policy</Text>
+    </Text>
+  );
+};
+
+export default TextTerms;
+
+const styles = StyleSheet.create({});
```

Comment:
This part of the styles remember to put it in another file or if you don't use it, delete it.

Reply:
Yes. thanks you, in next commit we'll correct this!

Reply:
8bcd3f67f0b33772e04824e6df56fd5877e5f8cc Done